src/components/Nav/NavList/NavList.js

```diff
@@ -50,11 +50,19 @@ export default class NavList extends Component {
   buildNewItemChild({ props }, index) {
     const { onItemClick, activeHref } = this.props;
 
+    const { onClick } = props;
```

Comment:
Could `onClick` have a propType definition? And also, what becomes of `onItemClick`? (Deprecated?)

Reply:
This is the `onClick` from specific `NavItem`s which never ended up being called - As far as I can tell `onItemClick` is a more general handler for all `NavItem`s within a `NavList`